cors/access-control-expose-headers-parsing.window.js

```diff
@@ -1,13 +1,15 @@
-function exposeTest(resource, desc) {
-  const url = new URL("resources/" + resource, location.href).href.replace("://", "://élève.");
+promise_test(() => fetch("resources/access-control-expose-headers.json").then(res => res.json()).then(runTests), "Loading JSON…");
 
-  promise_test(() => {
-    return fetch(url).then(res => {
-      assert_equals(res.headers.get("content-language"), "sure")
-      assert_equals(res.headers.get("x-custom"), null);
-    })
-  }, "Access-Control-Expose-Headers parsing: " + desc);
+function runTests(allTestData) {
+  allTestData.forEach(testData => {
+  	const encodedInput = encodeURIComponent(testData.input)
+    promise_test(() => {
+      const relativeURL = "resources/expose-headers.py?expose=" + encodedInput,
+            url = new URL(relativeURL, location.href).href.replace("://", "://élève.");
```

Comment:
Shouldn't this use the configurable domain rather than hardcoding it?

Reply:
Many years ago `*.<whatever>` was indeed required for wpt tests. Because we'd test any random domain, and that would be different for different tests.

Reply:
I was wrong on the `*.<whatever>`, I see there's just a small list of required domains, and élève.` is one of them. :)